tools/rosbag/include/rosbag/player.h

```diff
@@ -90,6 +90,7 @@ struct ROSBAG_DECL PlayerOptions
 
     std::vector<std::string> bags;
     std::vector<std::string> topics;
+    std::vector<std::string> pausetopics;
```

Comment:
Can you please update this name and all its usage including the command line to something including a separator: either `pause_topics` or `paused_topics`.